src/tests/components/SearchForm.test.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import SearchForm from "../../components/SearchForm";
+
+describe("SearchForm", () => {
+  const testFunction = () => {};
+  it("renders correctly", () => {
+    const { asFragment } = render(
+      <SearchForm
+        onSubmit={testFunction}
+        searchText=""
+        setSearchText={testFunction}
+      />
+    );
+    expect(asFragment()).toMatchSnapshot();
+  });
+  it("renders button text correctly", () => {
```

Comment:
Its also good to leave a clear line between each `it` block to help with readability